measures/apps/measuresdata/measuresdata.cc

```diff
@@ -244,8 +244,8 @@ const String &IERSpredict2000Format = IERSpredictFormat;
 
 // IGRF
 const String IGRFFormat = String("") +
-"a1 x2 i3 i3 f9 f9 f9 f9 f9 f9 f9 f9 f9 f9 f9" +
-" f9 f9 f9 f9 f9 f9 f9 f9 f9 f9 f9 f9";  // 22 coefficients and SV
+"a1 i3 i3 f7 f7 f7 f7 f7 f7 f7 f7 f7 f7 f7" +
+" f7 f7 f7 f7 f7 f7 f7 f7 f7 f7 f7 f7 f7 f7";  // 24 coefficients and SV
```

Comment:
The second line should be `" f7 f7 f7 f7 f7 f7 f7 f7 f7 f9 f10 f10 f9 f8";  // 24 coefficients and SV`